bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingFactoryHelper.java

```diff
@@ -237,7 +239,8 @@ public static Object getDefaultValueAsCorrectType(Type parameterType, String def
     /**
      * Apply the {@link ThingType}'s default values to the given {@link Configuration}.
      *
-     * @param configuration the {@link Configuration} where the default values should be added (may be null, but method
+     * @param configuration the {@link Configuration} where the default values should be added (may be null,
+     *            but method
```

Comment:
this could be combined with the next line.

Reply:
done